.gitignore

```diff
@@ -1 +1,51 @@
-/docs/build/
+# Files generated by invoking Julia with --code-coverage
```

Comment:
You want an actual .gitignore? Great. It helps a lot to have a clear `git status` when it really is clear.